app/assets/stylesheets/placeholder.scss

```diff
@@ -0,0 +1,25 @@
+.video--placeholder {
+  background-image: linear-gradient($pale-blue 100%, transparent 0); // blue rectangle at the top covering the video with color at a height of 100%
+  background-position: 0 0;     // left padding + top padding
+  background-repeat: no-repeat;
+  background-size: 100% 100%;   // width + height
+}
+
+.search-results--placeholder {
+  background-image: // background properties map the remaining placeholder shapes
+    linear-gradient($pale-blue 20%, transparent 0), // top line with color at a height of 15%
+    linear-gradient($pale-blue 15%, transparent 0), // middle line
+    linear-gradient($pale-blue 15%, transparent 0), // bottom line
+    linear-gradient($pale-blue 100%, transparent 0); // bottom right square
+  background-position: // background poistion maps to background-image
+    0 65%, // top line left padding + top padding
+    0 70%, // middle line
+    0 75%, // bottom line
+    100% 100%; // bottom right square
+  background-repeat: no-repeat;
+  background-size: // background-size maps to background-position
+    80% 20%, // top line width + height
+    80% 15%, // middle line
+    80% 15%, // bottom line
+    50% 25%; // bottom right square
+}
```

Comment:
There are a multitude of ways to do this and so I went for this for now at least. The rows are not the same height and currently the elements with-in the containers are of un-equal heights and not positioned every so using a % on the background meant I would get it as close to the individual height as device size changes etc: and to the placeholder wireframe for consistency. I understand what you are saying I don't think it will be consistent using the container elements as is right now at least. I think there is a story made since the load time changed that may help address this issue.

Reply:
I have added a setTimeout that removes the placeholder style and changed the video-container placeholder with a style in the _video.scss file